components/aggregation/mat/Form.js

```diff
@@ -148,24 +152,28 @@ export const Form = ({ onSubmit, testNames, query }) => {
           />
         </Box>
       </Flex>
-      <Flex justifyContent='space-between'  alignItems='center'>
-        <Box>
+      <Flex my={2} justifyContent='space-between' sx={{
+        '@media screen and (max-width: 476px)': {
+          flexDirection: 'column',
+        }
+      }}>
+        <Box width={[1, 0.18]}>
```

Comment:
Using `0.18` doesn't have the readability as `1/6th of the row`. The standard practice is to use fractions, so that the sibling Boxes can be configured similarly (e.g `2/6`, `3/6`). This ensures that the total width doesn't exceed  `1` (or `100%`).

Also in this case, I think making it `width={1/6}` would make the element too narrow for longer options like `HTTP Header Field Manipulation` to be hardly visible in screenshots etc. I would suggest using `1/5` because there are 5 boxes in the row anyway.

Reply:
The problem with `1/5` is that the width will become `20%` of all 5 elements.
That'll leave 0 space in between the elements, that's why I decided to use `18%` width which gave a nice spacing in between the elements. The `max-width` of the whole container is also `1280px` which is not giving enough space for the spacing.

How about using 9/50 instead of 0.18? 😂

Reply:
Usually I address that using padding within the elements. Although even that is can be a not-so-ideal solution at times.